simGL/simGL.py

```diff
@@ -91,6 +91,44 @@ def allelereadcounts_to_GL(Rg, e = 0.05):
                 GL.append(-np.log(np.power(((1-e)/2 + (e/3)/2), Rg[:, :, i]+Rg[:, :, j]) * 
                                   np.power(((e/3)/2 + (e/3)/2), Rg.sum(axis = 2)-Rg[:, :, i]-Rg[:, :, j])))
 
-    GL = np.array(GL)
-    return GL - GL.min(axis = 0)
+    GL = np.array(GL).transpose(1, 2, 0)
+    return GL - GL.min(axis = 2).reshape(GL.shape[0], GL.shape[1], 1)
+
+def allelereadcounts_to_pileup(allelereadcounts, filename = "tmp/reads.pileup"):
+    with open(filename, "w") as out:
+        first_line = True
```

Comment:
Unused variable? The `flake8` python package is good at finding these (among other things).